.github/workflows/sdk-update-test.yml

```diff
@@ -0,0 +1,148 @@
+        env:
+          NODE_OPTIONS: --unhandled-rejections=strict
+          FAUCET_SEED: ${{ secrets.PEREGRINE_FAUCET_SEED }}
+        run: |
+          yarn install
+          yarn run ts-node run_core_features.ts
+
+   create_pull_request:
+     runs-on: ubuntu-latest
+     needs: [workshop, core_features]
+     if: success()
+     steps:
+       - uses: actions/checkout@v1
+       - name: Get current package version
+         working-directory: code_examples/core_features
+         id: kiltprotocol_sdk
+         run: echo "::set-output name=current_tag::$(grep kiltprotocol/sdk-js package.json | awk -F \" '{print $4}')"
+       - uses: jacobtomlinson/gha-find-replace@v2
+         with:
+           find: '${{ steps.kiltprotocol_sdk.outputs.current_tag }}'
+           replace: '${{ github.event.client_payload.latestTag }}'
+           include: "docs/develop/03_workshop/**"
+           regex: true
+       - uses: jacobtomlinson/gha-find-replace@v2
+         with:
+           find: '@kiltprotocol/sdk-js": "${{ steps.kiltprotocol_sdk.outputs.current_tag }}'
+           replace: '@kiltprotocol/sdk-js": "${{ github.event.client_payload.latestTag }}'
+           include: "code_examples/**"
+           exclude: "**/*.lock"
+           regex: true
+       - name: Regenerate yarn lock
+         working-directory: code_examples/core_features
+         run: |
+           yarn install
+           yarn lint
+       - name: Regenerate yarn lock
+         working-directory: code_examples/workshop
+         run: |
+           yarn install
+           yarn lint
+       - name: Create Pull Request
+         uses: peter-evans/create-pull-request@v4
+         with:
+           token: ${{ secrets.REPO_ACCESS_TOKEN }}
+           base: develop
+           delete-branch: true
+           title: '[Chore] Latest sdk is compatible with docs'
+           body: |
+             Update sdk in the docs with the latest version ${{ github.event.client_payload.latestTag }}
+             - Beware ~ CI-generated PR
+           labels: |
+             sdk
+           draft: false
+           add-paths: |
+             *package.json
+             *yarn.lock
+             *.md
+   create_issue:
+     runs-on: ubuntu-latest
+     needs: [workshop, core_features]
+     if: failure()
+     steps:
+       - name: Checkout
+         uses: actions/checkout@v2
+       - name: Generate issue
+         run: |
+           cd .github/ISSUE_TEMPLATE
+           echo "[Either workshop or core_features test workflow](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}) using the latest sdk ${{ github.event.client_payload.latestTag }} on docs has failed.Please fix!" >> sdk-incompatible-with-docs.md
+       - name: Create issue
+         uses: peter-evans/create-issue-from-file@v3
+         with:
+           title: new sdk ${{ github.event.client_payload.latestTag }} is incompatible with docs examples
+           repository: KILTProtocol/ticket
```

Comment:
Yeah, I feel that is something we should discuss. Anyway, since broken docs are bad, perhaps I would add the high label since we are using the ticket board. Other than that, all the rest looks good 😃